### PR TITLE
Expose name and options on TranslationRepository

### PR DESCRIPTION
**Pull request by eputnam**
Would like to be able to access these things which would be helpful when creating repo chains. This adds an attribute accessor for both instance variables.

e.g.
```ruby
def self.chain_contains?(repository)
    @@translation_repositories.each do |in_chain|
      return true if repository.instance_variable_get(:@name) == in_chain.instance_variable_get(:@name)
    end
    false
end
```

### CODE REVIEW

File: lib/fast_gettext/translation_repository/base.rb [hunk 1]
```diff
@@ -4,6 +4,9 @@ module TranslationRepository
     #  - base for all repositories
     #  - fallback as empty repository, that cannot translate anything but does not crash
     class Base
+
+      attr_accessor :name, :options
```
Comment (grosser) [inline lib/fast_gettext/translation_repository/base.rb]: how about attr_reader ?

**Comment by grosser** [comment]
👍  looks good ... a test would be nice

**Comment by eputnam** [comment]
like so?

**Comment by grosser** [comment]
1.4.0

**Comment by eputnam** [comment]
awesome, thanks!

**Pull request merged**